types/jest-axe/index.d.ts

```diff
@@ -8,6 +8,10 @@
 
 import { AxeResults, Result, RunOnly } from "axe-core";
 
+declare module "axe-core" {
```

Comment:
Does `"axe-core"` have its own `Node` type? If you are stubbing the definition for `Node` from the DOM, then this should be in a global declaration:
```suggestion
declare global {
```

Reply:
`axe-core` doesn't have it's own `Node` type. However, by doing so, wouldn't `jest-axe` stub that for all of its dependents? We don't want that, right?

Reply:
If `"axe-core"` actually depends on a global `Node` and you create a forward declaration for `Node` _inside_ of `"axe-core"`, then it won't be for the actual global `Node` and will always be an empty interface (meaning just about anything would be assignment compatible to it).

If it is a forward declaration for a global type, it should be declared as a global. This is similar to what the NodeJS definitions currently do for forward declarations for other global types like `Iterable`.

Reply:
Makes sense!

Thanks for the explanation @rbuckton 👍